beacon-chain/state/stategen/setter_test.go

```diff
@@ -63,13 +62,16 @@ func TestSaveState_HotStateCanBeSaved(t *testing.T) {
 	}
 
 	// Should save both state and state summary.
-	if !service.beaconDB.HasState(ctx, r) {
+	_, e, err := service.epochBoundaryStateCache.getByRoot(r)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !e {
```

Comment:
Optional: I think e is too similar to "epoch" and "ok" seems more consistent with these type of existence checks. 
```suggestion
	_, ok, err := service.epochBoundaryStateCache.getByRoot(r)
	if err != nil {
		t.Fatal(err)
	}
	if !ok {
```